Formula/influxdb.rb

```diff
@@ -60,6 +60,56 @@ def install
     system "go", "build", *std_go_args(ldflags: ldflags), "-tags", "assets", "-o", bin/"influxd", "./cmd/influxd"
   end
 
+  def post_install
+    data = var/"lib/influxdb2"
+    data.mkpath
+
+    # Generate default config file.
+    config = etc/"influxdb2/config.yml"
+    unless config.exist?
+      config.write Utils.safe_popen_read(bin/"influxd", "print-config",
+                                         "--bolt-path=#{data}/influxdb.bolt",
+                                         "--engine-path=#{data}/engine")
+    end
```

Comment:
```suggestion
    config = buildpath/"config.yml"
    config.write Utils.safe_popen_read(bin/"influxd", "print-config",
                                       "--bolt-path=#{data}/influxdb.bolt",
                                       "--engine-path=#{data}/engine")
    (etc/"influxdb2").install config
```
`brew` never clobbers files that live in `etc`. If an existing one of the same name already exists, it will install the file it wants with the suffix `.default`.

Reply:
Hmmm seems `buildpath` is null/nil in `post_install`, is there another way for me to get a tmp path? I like the idea of having a `.default` file intelligently generated.

Reply:
Ah, I missed that this was in the `post_install`. Any reason we're not doing it in the `install` method? If it must be in `post_install`, we can use a [`Tempfile`](https://ruby-doc.org/stdlib-2.6.3/libdoc/tempfile/rdoc/Tempfile.html).

Reply:
I'll move it back to `install` 👍